extensions/ui/org.eclipse.smarthome.ui.basic/gulpfile.js

```diff
@@ -3,19 +3,19 @@
 
 	var
 		gulp = require("gulp"),
-		less = require("gulp-less"),
+		sass = require("gulp-sass"),
```

Comment:
This is a new tool that we didn't use before, correct? If so, we will require a CQ and approval by Eclipse.

Reply:
We could also stick with LESS unless it doesn't support some of the required features.

Reply:
I'd suggest to go with SASS because of the fact that MDL uses it natively by default. In the future we might extend/modify MDL stylesheets and possibly load them directly from `package.json` instead of hosting them as a static asset. 

See my comment from the first post:
> The reason for the last bullet point was to potentially utilize Material Design Lite in scss version only. That would give us some more flexibility wrt styling (e.g. extend css classes with internal mdl components or create your own).

I believe this approach will give us more flexibility overall and now is a good time to change it (since there's not much stylesheet to be maintained and conversion from LESS was pretty straightforward).

What's a process of submitting CQs? Is that something a PR creator should do?

Cheers! :-)

Reply:
@resetnow @kaikreuzer 🏓

Reply:
I think eventually having MDL building from sources is a good point.
 
>Is that something a PR creator should do?

No, as far as I understand, some people from Eclipse org can do it, including @kaikreuzer

Reply:
Correct, this has to be done by project committers.
Just to clarify: This is a build-time only dependency, i.e. we are not distributing anything from that project, right?

Reply:
@kaikreuzer that's right.